_posts/2024-05-02-EU-Digital-Assembly-2023.md

```diff
@@ -0,0 +1,61 @@
+
+This session featured four live demonstrations, all showcasing the cutting edge of European-based quantum technologies. 
+
+One of the demonstrations was presented by Dr. Miroslav Dobsicek, a senior research engineer at Chalmers University of Technology and a member of the NordIQuEst project team. Dr. Dobsicek's presentation focused on the practical aspect of current state quantum computing with superconducting circuits.
+
+{% include figure image_path="/assets/images/eu-digital-assembly-miroslav-presenting.png" alt="Screenshot of the Live Demonstration given at the EU Digital Assembly by Dr. Miroslav Dobsicek." caption="Miroslav Dobsicek presenting at the EU Digital Assembly 2023. Credit: Permanent Representation of Sweden to the European Union" %}
+
+He demonstrated the execution of a simple quantum computing application that harnessed the combined power of geographically separated computing resources.  
+
+This application utilized the LUMI high-performance computer (HPC) located in Finland for classical processing tasks, while simultaneously leveraging the processing power of the QAL 9000 quantum processor housed in Gothenburg, Sweden.  
+
+The project also utilized a pan-European approach to resource management, with project resources being allocated through the Puhuri allocation service in Estonia. 
+
+{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-lumi-puhuri-interconnection.png" alt="Diagram showing of the interconnection between QAL 9000, LUMI and Puhuri." caption="Interconnection between QAL 9000, LUMI and Puhuri. Credit: Dr. Miroslav Dobsicek" %}
+
+Dr. Dobsicek's live demonstration showcased two crucial stages involved in this innovative application.  
+ 
+At the outset, he demonstrated a live calibration measurement of the QAL 9000 processor. Quantum computing applications rely on incredibly precise control of quantum states.  
+
+To achieve this level of precision, Dr. Dobsicek utilized QAL 9000’s cloud RESTful API interface to perform a live recalibration measurement called a π-pulse. This specific pulse drives the transition of qubits, the quantum bits that hold information, between the quantum states of 0 and 1. With this recalibration a noticeable better state discrimination has been achieved.
+
+{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-live-tuneup.png" alt="A Screenshot of the live tune up of QAL 9000." caption="A Screenshot of the live tune up of QAL 9000. Credit: Martin Ahindura" %}
+
+Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on12 out of the 25 total qubits available on the QAL 9000 chip. 
+
+The state discrimination has improved after the recalibration; Less blue dots, |1> state, we present in the red point-could. 
```

Comment:
```suggestion
The state discrimination has improved after the recalibration; fewer blue dots, |1> state, we present in the red point-cloud. 
```

Reply:
I am not sure why there is a line in the page at this point.

Reply:
It seems that in this case jekyll does not like `|1>` even with Kramdown+Mathjax. Could you say "the first excited state" or the excited state instead?

Reply:
I could just do `|1>`